Resources/Locale/en-US/research/technologies.ftl

```diff
@@ -28,6 +28,7 @@ research-technology-uranium-munitions = Uranium Munitions
 research-technology-explosive-technology = Explosive Technology
 research-technology-weaponized-laser-manipulation = Weaponized Laser Manipulation
 research-technology-nonlethal-ammunition = Nonlethal Ammunition
+research-technology-energy-bola = Energy Bola
```

Comment:
Since the separate tech for energy bola was removed, there's no need for this locale.